test.yaml

```diff
@@ -0,0 +1,19 @@
+---
+
+# Note: First role has to initialize firewall_ports variable as empty array
+- name: Test port additions
+  hosts: test_host
+  vars:
+      firewall_ports: []
```

Comment:
We should probably have `roles/firewall/defaults/main.yaml` with this initialization, and have a separate `roles/firewall/commit` role that has the contents of your "post" role.  Then things that need firewall management would just need a dependency on the `firewall` role.